src/complate/index.js

```diff
@@ -19,15 +19,19 @@ export async function makeBundle(referenceDir) {
 }
 
 async function renderString(code, filename, context = {}) {
-	let { renderToString } = await load("rodunj/src/render", "complate extension");
+	let complate = await load("complate-ast", "complate extension");
+	let prefixed = {};
+	complate.runtimeSymbols.forEach(key => {
```

Comment:
Implementation note: I prefix the runtime dependencies of complate-ast to make collisions with user-defined identifiers less likely.

Reply:
This seems like it should be left to a module bundler: If runtime functionality resides in a dedicated module, all we need is an `import` statement and let the bundler take care of disambiguation? (That's also how Svelte does it AFAIK.)